app/cloud.go

```diff
@@ -0,0 +1,56 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+package app
+
+import (
+	"github.com/mattermost/mattermost-server/v5/model"
+)
+
+func (a *App) CheckAndSendUserLimitWarningEmails() *model.AppError {
+	if a.Srv().License() != nil && !*a.Srv().License().Features.Cloud {
+		// Not cloud instance, do nothing
+		return nil
+	}
+
+	// subscription, subErr := a.Cloud().GetSubscription()
+	// if subErr != nil {
+	// 	return subErr
+	// }
+
+	// if subscription != nil && subscription.IsPaidTier == "true" {
```

Comment:
Stripe returns the metadata as strings, I chose to not mess with the type of the return value throughout to prevent panics. Plus, the values for the configs we're checking in the front end are string true also - `license.Cloud === 'true'` etc. It could have gone either way, really.